### PR TITLE
Pig latin

### PR DESCRIPTION
**Pull request by eeshankuic**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks("grunt-contrib-jshint");
   grunt.loadNpmTasks("grunt-jscs");
 
-  var srcFiles = ["app/**/*.js", "server.js"];
+  var srcFiles = ["app/**/*.js", "server.js", "lib/**/*.js"];
```
Comment (stephanielingwood) [inline Gruntfile.js:5]: good catch!

File: app/script.js [hunk 2]
```diff
@@ -1,25 +1,61 @@
...
-}
+		//Request and then recieve reponse from server
+		$.get("/" + url, function(response) {
+			var resText;
 
-$("button").on("click", function() {
-  $("#display").text(randomIndexValue(stringArray));
-});
+			//Once received, format response basd on url requested
+			switch (url) {
```
Comment (stephanielingwood) [inline app/script.js:11]: interesting use of a switch case

File: lib/makePizza.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  //Convert pizza ingredients into a string
+  convertToPizzaString: function(crust, cheese, toppings) {
+    var pizzaString = crust + " pizza ";
+    pizzaString += "with " + cheese + " cheese ";
+    pizzaString += "and " + toppings;
+    return pizzaString;
+  },
+  //Search for an image via google
+  search: function(query, originalRes, callback) {
+    //Use request module to get img
```
Comment (EeshanKumar) [inline lib/makePizza.js]: This line to about line 32 are the lines that I took from the 'google-images' module. Re reading my code, I realize that's not clear.

File: lib/makePizza.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+module.exports = {
+  //Convert pizza ingredients into a string
+  convertToPizzaString: function(crust, cheese, toppings) {
+    var pizzaString = crust + " pizza ";
+    pizzaString += "with " + cheese + " cheese ";
+    pizzaString += "and " + toppings;
+    return pizzaString;
+  },
+  //Search for an image via google
```
Comment (eeshankuic) [inline lib/makePizza.js:14]: LIne 14 to line 32 are the lines that I took mostly from the 'google-images' module. Re reading my code, I realize that's not clear.

File: lib/makePizza.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  search: function(query, originalRes, callback) {
+    //Use request module to get img
+    request("http://ajax.googleapis.com/ajax/services/search/images?v=1.0&q=" + (query.replace(/\s/g, "+")) + "&start=0",
```
Comment (stephanielingwood) [inline lib/makePizza.js:17]: impressive! request is a powerful tool - glad you found it.

File: lib/makePizza.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
+//Code simplified and used from node module 'google-images'
+//https://github.com/vdemedes/node-google-images
+
+var request = require("request");
+
+module.exports = {
+  //Convert pizza ingredients into a string
+  convertToPizzaString: function(crust, cheese, toppings) {
```
Comment (stephanielingwood) [inline lib/makePizza.js:8]: love that you're exporting an object.

File: lib/makePizza.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    pizzaString += "with " + cheese + " cheese ";
+    pizzaString += "and " + toppings;
+    return pizzaString;
+  },
+  //Search for an image via google
+  search: function(query, originalRes, callback) {
+    //Use request module to get img
+    request("http://ajax.googleapis.com/ajax/services/search/images?v=1.0&q=" + (query.replace(/\s/g, "+")) + "&start=0",
+      function(err, res, body) {
+        //Assume no errors (I know, bad code, but I'm tired...)
```
Comment (stephanielingwood) [inline lib/makePizza.js:19]: it happens. It'll give you something to come back and work on. ;)

File: lib/piglatinify.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+  //Check if first letter is a vowel. If so, return new word
+  if (vowelHash.hasOwnProperty(wordArray[0])) {
+    wordArray[0] = wordArray[0].toUpperCase();
+    return wordArray.join("") + "-hay";
+  }
+
+  var count = 1;
+  //Check next two letters to see if they are vowels
+  while (count < 3) {
```
Comment (stephanielingwood) [inline lib/piglatinify.js:15]: good use case for a while loop

**Pull request merged**